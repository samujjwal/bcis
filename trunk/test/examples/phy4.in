#This is for Calculate the radius of orbit of the Moon
#Universal Gravitational consatant G=6.67*10^(-11)*Newton*meter^2/kilogram^2
#Earth Mass is 5.98E24 * kilogram
#Source from http://zebu.uoregon.edu/~probs/mech/grav/distmoon
#Test Program written by Changlong Jiang : cj2214@columbia.edu
#Date 12/15/2007

#load the pre-defined unit
load "si.phy"

#set variable
set x = 29.53 * day
nprint("seconds:",x)
print()
set Pi = 3.1415926
set G = 6.67E-11 * newton * (1*meter ^ 2) /(1*kilogram^2)

set masse = 5.98E24 * kilogram
set r = ((x*(1*G*(1*masse))^(1/2))/(2*Pi))^(2/3)
print(r)
print("Number is:", getNumber(r))
print("Unit is:",getUnit(r))

