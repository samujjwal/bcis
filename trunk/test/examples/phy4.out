seconds:2551392.0*second
Number is:2551392.0
Unit is:second
hours:708.72*hour
4.036521081066972E8*meter^1.0
Number is:4.036521081066972E8
Unit is:meter^1.0
